website/docs/tutorials/08-cairo-1-support/02-project-initialization.md

```diff
@@ -14,100 +17,159 @@ protostar init-cairo1 my-project
 After running the command, the following structure will be generated:
 
 ```
-my-project/
-├── src/
-│   └── main.cairo
-│   └── lib.cairo
+my_project/
+├── hello_starknet/
+│   ├── src/
+│   │   ├── business_logic/
+│   │   │   └── utils.cairo
+│   │   ├── contracts/
+│   │   │   └── hello_starknet.cairo
+│   │   ├── business_logic.cairo
+│   │   ├── contracts.cairo
+│   │   └── lib.cairo
 │   └── cairo_project.toml
 ├── tests/
-│   └── test_main.cairo
+│   ├── test_hello_starknet.cairo
+│   └── test_utils.cairo
 └── protostar.toml
 ```
 
-This template contains:
-
-- `src` directory
-    - `cairo_project.toml` which is needed for compilation 
-    - `main.cairo` file with one function definition
-    - `lib.cairo` file which defines the module
-- `test` directory
-    - Single test file with one test for function defined in `main.cairo`
-- `protostar.toml` containing information about the project
-
 :::warning
 This template will be changed in future versions, but the old one will still be usable with newer protostar versions
 :::
 
-## Cairo 1 modules
+### `hello_starknet`
 
-In order to understand how to create Cairo 1.0 modules, we need to talk about the purpose of `cairo_project.toml` and `lib.cairo`.
-### Project defaults
-#### 1. `cairo_project.toml`
-It is needed for the definition of crate roots, which are the places where `lib.cairo` files are located.
+This directory contains our only package in this project - `hello_starknet`.
 
-The default `cairo_project.toml` file contains only the definition of the `src` crate 
-```toml
-[crate_roots]
-src = "."
-```
+### `cairo_project.toml` and `lib.cairo`
 
-The `src` crate is then imported in our tests in the following manner:
-```
-use src::main::fib;
+All Cairo1 packages must define these files.
+
+- `cairo_project.toml` - this file defines the name of the package. It is good practice for this name to match the
+  top-level directory name - `hello_starknet` in our case.
+- `lib.cairo` - this file exposes all of our package to the compiler. Initially, it only contains two modules:
```

Comment:
you're kind of re-explaining them here. Instead i would suggest linking to the explanation and merging this explanation with the one from `understanding-cairo-packages`